ecosystem/typescript/sdk/src/generated/core/request.ts

```diff
@@ -12,6 +12,111 @@ import { CancelablePromise } from './CancelablePromise';
 import type { OnCancel } from './CancelablePromise';
 import type { OpenAPIConfig } from './OpenAPI';
 
+interface Cookie {
+  name: string;
+  value: string;
+  expires?: Date;
+  path?: string;
+  sameSite?: "Lax" | "None" | "Strict";
```

Comment:
should be in an ENUM

enum SITE = "Lax" | "None" | "Strict";

sameSite?: SITE

Reply:
sounds good. But I have to patch the openapi fork and create a new release. It is a bit painful. Next time when I patch it, will update this.

Reply:
I think perhaps the generator has an option to use enums, not sure though. We can also provide a custom request file for this, for which we don't need to update the fork either.